### PR TITLE
Migrate from Travis CI to GitHub Actions

### PR DESCRIPTION
**Pull request by vincenzobaz**

### CODE REVIEW

**Review by julienrf — COMMENTED** [review]
Nice! I will come back to this one after #1 is resolved, but the changes here look great to me!

File: .github/workflows/publish.yml [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+          SONATYPE_PASSWORD: ${{ secrets.SONATYPE_PASSWORD }}
+          SONATYPE_USERNAME: ${{ secrets.SONATYPE_USERNAME }}
+  publishSite:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: olafurpg/setup-scala@v10
+      - name: Generate site
+        run: sbt makeSite
+      - uses: JamesIves/github-pages-deploy-action@4.1.3
```
Comment (julienrf) [inline .github/workflows/publish.yml:28]: Interesting, so we don’t need sbt-ghpages anymore, and we don’t need to create special user credentials as explained here? https://docs.scala-lang.org/overviews/contributors/index.html#continuous-publication

**Review by julienrf — APPROVED** [review]
This looks good to me, but I guess we have to merge it to test it?

**Comment by julienrf** [comment]
Also, what should we do with this setting? https://github.com/scalacenter/library-example/blob/4d35fdef8e73a4d1b5300670cc496b7436969e63/build.sbt#L22

File: .github/workflows/publish.yml (hunk 1 above)
Comment (vincenzobaz) [reply .github/workflows/publish.yml:28]: Yes that is correct. GitHub exposes tokens to the action that can use it to interact with the repo.

Comment (julienrf) [reply .github/workflows/publish.yml:28]: Excellent news! Then I guess we can safely remove the sbt-ghpages plugin from this repo?

**Comment by vincenzobaz** [comment]
If we maintain those lines in the author's guide, we should keep them here too I think

**Review by adpi2 — APPROVED** [review]
LGTM!

**Comment by julienrf** [comment]
> If we maintain those lines in the author's guide, we should keep them here too I think

The idea is to update the lib author guide accordingly, no?

**Comment by vincenzobaz** [comment]
What I meant is that this repo and that guide should be coherent. We can keep those lines or drop them, I do not have strong opinions, as long as we do it in both places.

**Comment by julienrf** [comment]
I think we should only keep them if they are useful. Otherwise, the simpler the better!

**Pull request merged**